MonMaraicher/Features/Map/FarmerService.swift

```diff
@@ -7,21 +7,27 @@
 
 import Foundation
 
+enum LoadFarmerError: Error {
```

Comment:
`FarmerServiceError` is more representing your need IMO. Mostly because you won't create a specific Error enum for each func in your service.

Suggestion / tips:

use the namespacing feature to declare your error like that:
```
extension FarmerService {
  enum Error: Swift.Error {....}
 ```
 Because your error only belongs to your service it could be keep inside it. Up to you to do that ofc ;)